docs/HACKING.md

```diff
@@ -212,41 +267,85 @@ opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$
 ```
 
 You should then push the newly created index image to the remote repository:
+
 ```shell script
 podman push $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
-#### Editing an existing operator index
+##### Editing an existing operator index
+
 An existing operator index can have bundles added to it:
+
 ```shell script
 opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
+
 and removed from it:
+
 ```shell script
 opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
-## Updating Git submodules
+
+## Test
```

Comment:
This is odd. Why is this a standalone `Test` sub-heading. This is just `Deploying a workload`. Please stop saying `custom workloads` because there is no non-custom workload.